photo_points/Views/Admin/PhotoStream.cshtml

```diff
@@ -1,44 +1,39 @@
 ﻿@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
-@model IEnumerable<photo_points.Models.PhotoPoint>
+@model photo_points.ViewModels.PhotoStreamViewModel;
 @{
     ViewData["Title"] = "Welcome";
     Layout = "../Shared/_Layout.cshtml";
 }
-
+<script type="text/javascript" src="~/js/captureStatusButtons.js"></script>
 
 <h2>PhotoStream</h2>
 
-@foreach (var photo in Model)
-{
-
-    <div class="container card-body-sm" data-toggle="modal" data-target="#exampleModal">
-        <div class="row">
-            <img class="card-img-sm" src="~/images/fern.jpg" />
-        </div>
-        <div class="row card-sm-content">
-            <h3>@photo.locationName</h3>
-            <div class="col-sm-10">
-                <div class="row">
-                    <a asp-action="DeleteFromPhotoStream" asp-controller="Admin"
-                       asp-route-id="@photo.photoPointID">Remove</a>
-                    <h6>Jan 15, 2013</h6>
+<form asp-action="PhotoCard" 
+      asp-controller="Admin">
+       
+    @foreach (Capture capture in @Model.ApprovedCaptures)
+        {
+            <div class="container card-body-sm" capture-card-id="@capture.captureID">

```

Comment:
If this is truly a copy of the card html in Pending.html, you should put the code for the card into its own cshtml page somewhere that accepts a Capture as a model, and then in both files, replace the card html with @Html.Partial("pagename", model);

Reply:
I'm not sure I understand. I should make a third cshtml page and then make Pending and PhotoStream both reference that third page to save on duplicate code? So we could even make more pages later on using the same card html? I like that idea but want to make sure I understand you right

Reply:
yep thats what i meant